pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/HybridClusterIntegrationTest.java

```diff
@@ -301,7 +295,9 @@ public Boolean apply(@Nullable Void aVoid) {
     stopServer();
     stopBroker();
     stopController();
-    KafkaStarterUtils.stopServer(_kafkaStarter);
+    for (KafkaServerStartable kafkaStarter : _kafkaStarters) {
```

Comment:
Can we wrap this code as `stopKafka()`, just like `startKafka()`?

Reply:
Done